src/router.ts

```diff
@@ -0,0 +1,8 @@
+import { Router } from 'express';
+import traineeRouter  from './controllers/trainee/routes';
```

Comment:
This import is correct., But you can import directly from "./controllers/trainee" else there is no use of file "./controllers/trainee/index.ts"